.bazelrc

```diff
@@ -1,11 +1,11 @@
 # A few settings for XCHammer's .bazelrc
 # Debugging: Use the remapping feature of rules_swift.
 build \
- --spawn_strategy=standalone \
+ --spawn_strategy=local \
  --strategy=SwiftCompile=worker \
  --swiftcopt=-Xwrapped-swift=-debug-prefix-pwd-is-dot \
  --experimental_strict_action_env=true
 
 test \
- --spawn_strategy=standalone \
+ --spawn_strategy=local \
```

Comment:
spawn_strategy `standalone` is deprecated. Docs say "Please use local instead."

https://docs.bazel.build/versions/master/user-manual.html#flag--spawn_strategy